src/main/java/seedu/address/model/pet/Age.java

```diff
@@ -30,20 +30,21 @@ public Age(int age) {
      * @param pet The pet that needs to be updated the age.
      * @return Age in integer
      */
-    public int getPetAge(Pet pet) {
+    public static Age generateAge(Pet pet) {
         requireNonNull(pet);
         LocalDate currentDate = LocalDate.now();
         LocalDate dateOfBirth = pet.getDateOfBirth().getDate();
         requireAllNonNull(currentDate, dateOfBirth);
-        return Period.between(dateOfBirth, currentDate).getYears();
+        int result = Period.between(dateOfBirth, currentDate).getYears();
+        return new Age(result);
     }
 
     /**
      * Gets the age.
      * @return Age in integer
      */
-    public int getAge() {
-        return age;
```

Comment:
Can you update the method name in the Age test so that the test case will pass and CI will pass? thanks!